lib/matplotlib/axis.py

```diff
@@ -1097,6 +1108,9 @@ def get_tightbbox(self, renderer):
         else:
             return None
 
+    def get_tick_padding(self):
+        return self.majorTicks[0].get_tick_padding()
```

Comment:
Another point: are we always guaranteed at least one tick?

On Thu, Dec 17, 2015 at 11:59 AM, Michael Droettboom <
notifications@github.com> wrote:

> In lib/matplotlib/axis.py
> <https://github.com/matplotlib/matplotlib/pull/5683#discussion_r47930730>:
>
> > @@ -1097,6 +1108,9 @@ def get_tightbbox(self, renderer):
> >          else:
> >              return None
> >
> > +    def get_tick_padding(self):
> > +        return self.majorTicks[0].get_tick_padding()
>
> That's a good point.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5683/files#r47930730>.
>